### DISCUSSION TITLE
Is there a Figma ?

### DISCUSSION THREAD

**Discussion by ossamabenjemaa** [amnestywebsite/humanity-theme#351] [Q&A] [answered]
Hello,

Has a Figma file been created? It would be great to have one. Thanks!

Feel free to tweak it further if needed!

**Comment by AmnestyAM** [comment, marked as answer]
Hello @OuzeOzam thanks for getting in touch with us, unfortunately not. Are you using the theme or are you still considering if you should use it or not?

**Reply by ossamabenjemaa** [reply]
Hi there! Yes, my team and I plan to use it and potentially create an evolving and scalable design system in Figma. Are you interested in collaborating?

**Reply by AmnestyAM** [reply]
That is amazing news, glad to hear you will be using the theme. Welcome! 

@carolinejcourtney did you have any thoughts on this?

**Reply by carolinejcourtney** [reply]
Welcome @OuzeOzam! This sounds great, let me know what I can do to help!

**Reply by ossamabenjemaa** [reply]
Hi, yeah i'll let you know!
